.github/workflows/php-compatibility.yml

```diff
@@ -0,0 +1,55 @@
+name: PHP Compatibility
+
+env:
+  COMPOSER_VERSION: "2"
+  COMPOSER_CACHE: "${{ github.workspace }}/.composer-cache"
+
+on:
+  push:
+    branches:
+      - develop
+      - trunk
+  pull_request:
+    branches:
+      - develop
+
+jobs:
+  php_compatibility:
+    name: ${{ matrix.php }}
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        php: [ '5.6', '7.0', '7.4', '8.0', '8.1' ]
```

Comment:
You can remove this matrix here and from the places where it is used. See this [Slack post](https://10up.slack.com/archives/C84M9AA06/p1648460113851159) for more details.